requirements/django.txt

```diff
@@ -0,0 +1 @@
+django==2.2.11            # via -r requirements/base.in, django-crum, django-model-utils, djangorestframework, drf-jwt, edx-django-utils, edx-drf-extensions, rest-condition
```

Comment:
Can you explain why you need this file?  When I removed the constraint in `base.in`  then the tox setup worked fine for me without this addition.

Reply:
Mainly to be precise about the exact version being tested.  This also keeps development environments up to date as new Django point releases come out.  In services it's more important, since it enforces that we've tested with the exact version being deployed.